src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,158 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import React, { useState, useCallback } from 'react';
+import { getTodos, getFilteredTodos } from '../../api';
+import { Todo } from '../../types/Todo';
+import { Selected } from '../../types/Enum';
+
+type Props = {
+  setTodos: (value: Todo[]) => void,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  setTodos,
+}) => {
+  const [selectedOption, setSelectedOption] = useState('all');
+  const [value, setValue] = useState('');
+
+  const getIncludesTodos = useCallback(
+    (val: string, currTodos: Todo[]) => {
+      return currTodos.filter((todo) => todo.title.includes(val));
```

Comment:
I found the problem!

EXAMPLE:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/107731333/f7bfd25a-d54a-476e-9881-08a70a194d1a)

![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/107731333/4258e972-9b68-43e9-83ba-d95ca4777ea9)

Our query should ignore cases, so that's the problem!

Reply:
Thanks, try to do)